CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+
+## @rjsf/utils
+- Fix data type in `FieldTemplateProps['onChange']`
+
+
```

Comment:
You can just move this down under the `@rjsf/utils` below in `5.13.1` since it has yet to be published... I.e.:

```
## @rjsf/utils
- Added `getOptionMatchingSimpleDiscriminator()` function
- `getMatchingOption` and `getClosestMatchingOption` now bypass `validator.isValid()` calls when simple discriminator is provided, fixing [#3692](https://github.com/rjsf-team/react-jsonschema-form/issues/3692)
- Fix data type in `FieldTemplateProps['onChange']`
```

Reply:
I thought it was already published! Fixed now.